src/lightning/data/streaming/reader.py

```diff
@@ -93,10 +81,17 @@ def _delete(self, chunk_index: int) -> None:
 
     def stop(self) -> None:
         """Receive the list of the chunk indices to download for the current epoch."""
-        self._to_stop_queue.put(True)
+        self._to_download_queue.put(_END_TOKEN)
 
     def _maybe_delete_chunks(self) -> None:
-        chunk_index = _get_from_queue(self._to_delete_queue)
+        reached_pre_download = self._pre_download_counter == self._max_pre_download
+
+        # we have already pre-downloaded some chunks, we just need to wait for them to be processed.
+        # Note: The timeout here should not be too short. We need to prevent the caller from aggressively
+        # querying the queue and consuming too many CPU cycles.
```

Comment:
Can you move this comment to where the timeout is defined please? The comment is otherwise useless if it is not right next to the number.

Reply:
done !